tests/test_operation.py

```diff
@@ -58,3 +58,10 @@ def test_params_setter(server):
     assert params['param1'] == 'bar'
     assert params['param2'] == 'bar'
     assert params['param3'] == 'plop'
+
+
+def test_context_setter(server):
```

Comment:
This is almost good :)
Could you update to use this code:
```diff
diff --git a/tests/test_operation.py b/tests/test_operation.py
index 9c6d806..e614cf8 100644
--- a/tests/test_operation.py
+++ b/tests/test_operation.py
@@ -63,5 +63,6 @@ def test_params_setter(server):
 def test_context_setter(server):
     operation = server.operations.new('Noop')
     operation.context = {'currentDocument': 'foo'}
-    context = operation.context
-    assert context['currentDocument'] == 'foo'
+    with pytest.raises(HTTPError):
+        operation.execute()
+    assert operation.context['currentDocument'] == 'foo'
```
Like that the `build_payload()` method will be entirely covered.

Reply:
Or even better, I would be happier to merge a complete test reflecting your use case (using `FileManager.Import` for example, reduced to the minimum). Do you think it is possible?

Reply:
I am going to apply the simple change. Not sure when I find time for the additional test.